### ISSUE TITLE
Related Content: mismatch information Calypso/WP Admin

### ISSUE THREAD

**Issue by cecilearkay** [Automattic/wp-calypso#41730]
The Related Content section in WP Admin does not match what's on Calypso, causing confusion as to which options are actually available.

#### Steps to reproduce

* On WP Admin > Settings > Reading, you can choose to display a thumbnail, the "context" (tags/categories) and the publishing date.

* On Calypso > Tools > Marketing > Traffic, you only have the option to display a thumbnail. The "context" is displayed by default and can't be removed. There's no option to display the publishing date.

#### What I expected

I expected the information to match between both dashboards, or at least WP Admin to mention the Related Content section is outdated and must be managed on Calypso.

#### What happened instead

WP Admin does not mention the Related Section is outdated, and the context/publishing date options **do not render on the front end**. Only Calypso’s options works.

**Comment by jordesign** [comment]
Adding some screenshots for reference:

WP Admin:
<img width="1267" alt="Screen Shot 2021-02-03 at 11 41 37 am" src="https://user-images.githubusercontent.com/1842363/106681605-f1615900-6614-11eb-9f5c-9f3b399e70fd.png">

Calypso:
<img width="1043" alt="Screen Shot 2021-02-03 at 11 42 27 am" src="https://user-images.githubusercontent.com/1842363/106681638-fd4d1b00-6614-11eb-80b2-d578f974c391.png">

**Comment by obenland** [comment]
Let's remove the wp-admin settings. 

/cc @jeherve In case you have more history or context for why this is duplicated and possible reasons to keep it that way.

**Comment by jeherve** [comment]
> In case you have more history or context for why this is duplicated and possible reasons to keep it that way.

We've gone through different strategies over the years:

- When Jetpack was first released, its settings were scattered across core settings pages in wp-admin, to blend in with the rest of the core WP settings. This made sense on WordPress.com to avoid confusing site owners.
- Over time, we decided to also add those settings to the React dashboard in wp-admin, as a way to show folks the value of Jetpack, and make it clearer what features were added by Jetpack.
- When Calypso came along, we aimed to have all settings (whether they be from the wp-admin React dashboard, from core WP, or from Jetpack added to core WP settings) available in Calypso as well.

As a result, we end up with some settings available in multiple areas today. Related Posts are even worse because we also have settings in the Customizer for those same options.

In short, I think it would be fine to remove the settings from wp-admin > Settings > Reading at this point, given that the settings are also available in the customizer, and that we link to the customizer from the wp-admin Jetpack React dashboard.
